### ISSUE TITLE
Links added to sidebar have different widths

### ISSUE THREAD

**Issue by EfficiencyVI** [NabokD/pockettube#111]
Hi,

just a minor problem for the users with OCD. The link to the deck and the group entries have different width.

![Screenshot_20240727_014001](https://github.com/user-attachments/assets/15cf43b8-89b3-4a9c-b028-a2f26045f6e5)

```
.ysm-deck-open {
    width: calc(100% - 12px);
}
ytd-guide-section-renderer {
    width: calc(100% - 24px);
}
```

added in build/context.css fixes both. Love your addon! ❤️

**Comment by NabokD** [comment]
Thank you for sharing! I'll include it in the next release!

**Comment by NabokD** [comment]
It has been included in version 16.2.7
